.dev/script/hk_accidents - description for categorical variables.R

```diff
@@ -0,0 +1,70 @@
+# Load packages-----------------------------------------------------------------
+library(tidyverse)
```

Comment:
Wow! Great work @gabtam55 ! Two comments:

- The documentation for the variables (i.e. what you see when you run `?hk_accidents`) may be changed in the corresponding R file in the `R` directory. We can add those in a separate pull request. 
- For best practice, we may decide to only load packages that are _minimally necessary_, e.g. **dplyr** and **tidyr**, rather than the entire **tidyverse**. It matters less for what we're doing here but it will be better in the long term if there is a need to reproduce what we've got :smile:

Reply:
Gotcha for both feedback. Will do next time :)